src/encoded/schemas/analyte.json

```diff
@@ -219,6 +337,24 @@
                 "type": "string",
                 "linkTo": "Sample"
             }
+        },
+        "total_yield": {
+            "title": "Total Yield",
+            "description": "Total yield of analyte from sample (ug)",
+            "type": "number",
+            "minimum": 0
+        },
+        "yield_per_million_cells": {
+            "title": "Yield per million cells",
+            "description": "Yield per million cells (ug/M cells), if applicable",
+            "type": "number",
+            "minimum": 0
+        },
+        "yield_per_mg_tissue": {
+            "title": "Yield per mg tissue",
+            "description": "Yield per milligram of tissue (ug/mg), if applicable",
+            "type": "number",
+            "minimum": 0
```

Comment:
Not necessary here, as I doubt this data will be searched on, but can always separate out the number from the units. The benefit of the separation approach is that if someone wants to provide different units later on, it's typically a simple addition to the unit enum, whereas the approach above forces either a redesign or more properties to capture essentially identical information. Also, when the field is split between number and unit, search is much more direct.

Reply:
I've changed to split fields between number and unit.